src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -339,6 +339,11 @@ extension AppDelegate: ENATaskExecutionDelegate {
 			return
 		}
 
+		self.taskScheduler.notificationManager.presentNotification(
+			title: "BGTask event received",
+			body: "\(ENATaskIdentifier.detectExposures.backgroundTaskSchedulerIdentifier)",
+			identifier: ENATaskIdentifier.detectExposures.rawValue)
+
```

Comment:
Not really part of this PR?

Reply:
Shouldn't have been committed, was used for debugging the BGTasks